Misc/NEWS.d/next/Library/2021-02-09-10-06-37.bpo-43178.Lbvvsz.rst

```diff
@@ -0,0 +1 @@
+Removed redundant method definitions of _qsize in PriorityQueue and LIFOQueue. These methods already inherit the exact same implementation from their parent class, Queue, so these are not necessary
```

Comment:
Without considering the code itself, there is no need for a NEWS entry as this change is not user-facing.

Reply:
Sorry. This is my first pull request, I have been trying to follow the tutorial closely. I will pay more attention in the future

Reply:
Do I need to fix this?

Reply:
> Sorry. This is my first pull request, I have been trying to follow the tutorial closely. I will pay more attention in the future

Is fine, don't worry :)

Normally you need to add a NEWS file, but this is one of the cases were this is not needed.

>Do I need to fix this?

Yeah, please, delete this file